src/js/components/Layer/LayerContainer.js

```diff
@@ -77,6 +86,28 @@ const LayerContainer = forwardRef(
     }, [position, ref]);
 
     useEffect(() => {
+      const onClickDocument = event => {
+        // determine which portal id the target is in, if any
+        let clickedPortalId = null;
+        let node = event.target;
+        while (clickedPortalId === null && node !== document) {
```

Comment:
should check for "" on getAttribute: https://developer.mozilla.org/en-US/docs/Web/API/Element/getAttribute

Reply:
Done.